frontend/src/pages/RegionalDashboard/components/RecipientSpotlightDashboardCards.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import WidgetH2 from '../../../components/WidgetH2';
+import Container from '../../../components/Container';
+import NoResultsFound from '../../../components/NoResultsFound';
+
+export default function RecipientSpotlightDashboardCards({
+  recipients,
+  loading,
```

Comment:
the idea behind not using a loading state in useFetch and using the context was that we wouldn't have all these little loading indicators on each widget

Reply:
oops thought i removed it no need for two loaders.